app/policies/work_policy.rb

```diff
@@ -9,14 +9,17 @@ class WorkPolicy < ApplicationPolicy
     scope.where(depositor: user)
   end
 
-  # Only administrators or depositors in a specific collection are able to create new collection members
+  # Can deposit a work iff:
+  #   1. Collection is accessioned
+  #   2. The user is an administrator, or a depositor or a manager of this collection
   sig { returns(T::Boolean) }
   def create?
+    collection = record.collection
+    return false unless collection.accessioned?
```

Comment:
Do we need a new work policy spec to cover this or does the current set of specs already cover this?

Reply:
I believe it's already covered in the work_policy_spec.rb